### PR TITLE
Use steps when moving the range slider thumb

### LINKED ISSUE #429: Range slider steps are not working
### Context

When creating a `<as-range-slider>` component and setting the `step` value, it is not applied to the component

### Steps to Reproduce

1. Add `<as-range-slider id="simple" step="5" min-value="5" max-value="30"></as-range-slider>`
2. Drag the slider

### Current Result

It does not move the slider 5 by 5, but 1 by 1 instead

### Expected result

It should apply the `step` value properly

### Browser and version

Chrome

### PR DESCRIPTION
**Pull request by rjimenezda**
Fix #429
---

It was working properly for keyboard events and moving the bar, but not for moving the thumb.

### CODE REVIEW

**Comment by IagoLast** [comment]
(update tests)

**Comment by rjimenezda** [comment]
They actually are, but there were no specs for steps (I'm on it though)

**Comment by elenatorro** [comment]
💖

File: packages/components/src/components/as-range-slider/slider/as-range-slider.tsx [hunk 1]
```diff
@@ -304,23 +304,10 @@ export class RangeSlider {
   }
 
   private _getStepValue(value) {
-    const stepValue = (value / this.step) * this.step;
+    const stepValue = Math.round(value / this.step) * this.step;
```
Comment (VictorVelarde) [inline packages/components/src/components/as-range-slider/slider/as-range-slider.tsx]: 👍 great catch! @rjimenezda

**Pull request merged**